.github/workflows/wheels.yml

```
@@ -98,6 +98,21 @@ jobs:
           name: ${{ matrix.python[0] }}-${{ startsWith(matrix.buildplat[1], 'macosx') && 'macosx' || matrix.buildplat[1] }}
           path: ./wheelhouse/*.whl
 
+      # Used to push the built wheels
+      # TODO: once Python 3.11 is available on conda, de-dup with
+      # setup python above
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-update-conda: true
+          # Really doesn't matter what version we upload with
+          # just the version we test with
+          python-version: '3.8'
+
+      - name: Install anaconda client
+        if: ${{ success() && (env.IS_SCHEDULE_DISPATCH == 'true' || env.IS_PUSH == 'true') }}
+        run: |
+          conda install -q -y anaconda-client

```

I assume you know, but for one liners you can also use this:

```suggestion
        run: conda install -q -y anaconda-client

```